eng/common/scripts/Cadl-Project-Generate.ps1

```diff
@@ -11,48 +11,10 @@ param (
 
 $ErrorActionPreference = "Stop"
 . $PSScriptRoot/Helpers/PSModule-Helpers.ps1
+. $PSScriptRoot/Helpers/CommandInvocation-Helpers.ps1
```

Comment:
I don't believe this script is used by anyone any longer.

Reply:
Changes removed from this PR